src/main/java/com/revature/ams/AirportRunner.java

```diff
@@ -25,7 +38,12 @@ public static void main(String[] args) {
             System.out.println("1. Flight Information"); // ctrl + d on a line will repeat
             System.out.println("2. Add Flights");
             System.out.println("3. Update Flight"); // sout - shorthand for System.out.println();
-            System.out.println("4. Exit"); // sout - shorthand for System.out.println();
+            // TODO: REVIEW ME
+            System.out.println("4. Register");
```

Comment:
why not stop at the register and create a second scanner with the options under register. sort of like when you call into an automated system.

Reply:
Great question, the AirportRunner is starting to get really bloated with options & selections. However, this is to offer you all a means to visualize the concept of a FrontController for HTTP Request & Response once we get to Javalin. Our AirportRunner is acting as a FrontController to determine what model-specific controllers are required based on the users request.